cmd/kind/kind.go

```diff
@@ -74,13 +75,13 @@ func NewCommand() *cobra.Command {
 
 func runE(flags *Flags, cmd *cobra.Command, args []string) error {
 	level := defaultLevel
-	parsed, err := log.ParseLevel(flags.LogLevel)
+	parsed, err := logrus.ParseLevel(flags.LogLevel)
```

Comment:
This continues to expose logrus directly. Shouldn't that be an implementation detail of pkg/log and abstracted here?

Reply:
ack